code/modules/projectiles/projectile/bullets/special.dm

```diff
@@ -21,7 +21,11 @@
 	. = ..()
 	var/mob/M = target
 	if(istype(M))
-		M.slip(100, M.loc, GALOSHES_DONT_HELP|SLIDE, 0, FALSE)
+		if(M.anti_magic_check())
+			src.impact_effect_type = null
```

Comment:
you don't need this src

Reply:
tbh i dont need the entire line, it doesnt do anything since the impact effect fires before it changes, thankfully the banana impact effect just looks like a richochet already so its not a problem. it is a problem if i wanna do the same thing with the mime gun bullets since those impacts depict blood visually